src/vmm_mad/exec/one_vmm_exec.rb

```diff
@@ -1294,6 +1304,7 @@ def destroy(id, drv_message, signal)
 threads            = 15
 shell              = 'bash'
 local_actions      = {}
+delegate_actions   = {}
```

Comment:
Maybe have a default here which contains `migrate` action?

(the other option would be to update oned.conf and add migrate into arguments
`ARGUMENTS      = "-d migrate -t 15 -r 0 kvm",`)